src/app/Main/Body/QueryPage/Results/Charts/ApexChart.model.ts

```diff
@@ -1,7 +1,7 @@
 export type ApexChartData = {
     xLabels: string[];

```

Comment:
so consider rename this to, to adopt the new names, e.g. dataLabels

Reply:
Done. Commit "Little Changes"